radio/src/tasks_arm.cpp

```diff
@@ -91,6 +91,32 @@ uint16_t stackAvailable()
 }
 #endif
 
+uint32_t timeForcePowerOffPressed = 0;
```

Comment:
Should be `volatile`. It could also be `uint16_t` to save 2 bytes of RAM, still enough range (wrap around in 650 seconds).

```c
volatile uint16_t timeForcePowerOffPressed;
uint16_t delay = (int16_t)get_tmr10ms() - timeForcePowerOffPressed;
```